cit3.0-web/src/components/OpportunitiesListContainer/OpportunitiesListContainer.js

```diff
@@ -4,28 +4,39 @@ import { Row } from "react-bootstrap";
 import axios from "axios";
 import OpportunityList from "../OpportunityList/OpportunityList";
 import Paginator from "../Paginator/Paginator";
+import OpportunityListItem from "../OpportunityListItem/OpportunityListItem";
 
-export default function OpportunityListContainer({ totalCount }) {
+export default function OpportunityListContainer({
+  totalCount,
+  component: Component,
+  query,
+}) {
   const [opportunities, setOpportunities] = useState(null);
   const [currentPage, setCurrentPage] = useState(1);
+  // eslint-disable-next-line
```

Comment:
Can we remove this? I can't see what issues eslint would be having here

Reply:
Just an unused pagination thing, I needed to get around lint for now, as Sarah is working in there.